csfieldguide/templates/interactives/short-term-memory-picture.html

```diff
@@ -0,0 +1,125 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load i18n svg %}
+{% load static %}
+
+{% block html %}
+  <div class="container">
+    <div>
+      <h3 class="text-center">{% trans "Short term memory picture game" %}</h3>
+      <p class="text-center intro-content">{% trans "Try to remember as many items as you can without writing anything down in 30 seconds.<br>When you are ready, click the button below to display the items!" %}</p>
```

Comment:
How about: `What items can you remember in 30 seconds?<br>Don't write them down!` or similar